lib/open_food_network/order_cycle_permissions.rb

```diff
@@ -90,8 +90,8 @@ def visible_variants_for_incoming_exchanges_from(producer)
         Spree::Variant.joins(:product).where('spree_products.supplier_id = (?)', producer)
       else
         # All variants of the producer if it has granted P-OC to any of my managed hubs that are in this order cycle
-        permitted = EnterpriseRelationship.permitting(managed_participating_hubs).
-          permitted_by(producer).with_permission(:add_to_order_cycle).present?
+        permitted = EnterpriseRelationship.permitting(managed_participating_hubs.select(:id)).
```

Comment:
Interesting, so you can pass a relation as enterprise ids in `where('child_id IN (?)', enterprise_ids)` and it gets executed as subquery? That's awesome!